app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -291,4 +291,64 @@ public function validateBeforeOrderCommit($observer) {
         $orderModel = Mage::getModel('boltpay/order');
         $orderModel->validateBeforeOrderCommit($observer);
     }
+
+    /**
+     * Automatically create an invoice after creating the shipment if createInvoiceAfterCreatingShipment setting is set to yes
+     * @param $observer
```

Comment:
@johnmccombs1  I find it useful to know the object type and in this case to know which elements are stored in the observer (i.e. a Magento Shipment model that has a specific class name.)